packages/server/src/datasources/dataset-service.ts

```diff
@@ -86,10 +85,18 @@ export class DatasetService extends DataSource {
   }
 
   async createDataset(classes: string[], maxImages: number): Promise<CreateJob> {
-    const id = uuidv4();
-    const directory = `data/create/${id}`;
+    const dataset = Dataset.build({ name: classes[0] });
+    const directory = `data/create/${dataset.id}`;
     await mkdirp(directory);
-    return await this.mLService.create(classes, maxImages, directory, id);
+    await mkdirp(`data/datasets/${dataset.id}`);
+    const createJob = await this.mLService.create(classes, maxImages, directory, dataset.id);
+    const name = createJob.zipPath.split("/")[1]; // Axon_Dataset_ETC.zip
+    dataset.name = name; // Axon_Dataset_ETC.zip
```

Comment:
Playing with this more and more, I think having the dataset name be something more like OpenImages_classname might be better?

Reply:
`OpenImages_{class name}.zip` is now the .zip name.